Sources/StreamChat/Workers/MessageUpdater.swift

```diff
@@ -241,19 +241,22 @@ class MessageUpdater<ExtraData: ExtraDataTypes>: Worker {
     ///   - score: The reaction score.
     ///   - extraData: The extra data attached to the reaction.
     ///   - messageId: The message identifier the reaction will be added to.
+    ///   - enforceUnique: If set to `true`, new reaction will replace all reactions the user has (if any) on this message.
     ///   - completion: Called when the API call is finished. Called with `Error` if the remote update fails.
     func addReaction(
         _ type: MessageReactionType,
         score: Int,
         extraData: ExtraData.MessageReaction,
         messageId: MessageId,
+        enforceUnique: Bool = false,
```

Comment:
This is an updater, so the default parameter here is not needed. Did I miss something or is the public API for this feature missing? :)

Reply:
Idk how this happened and don't have an excuse 😳
Fixed!